drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -120,6 +123,17 @@ public void setChromeOptions(WebDriverConfiguration configuration, DesiredCapabi
         String browser = configuration.getBrowser().toLowerCase();
         if (browser.equals("chromeheadless")) {
             chromeOptions.addArguments("--headless");
+            if (configuration.getDimensions() != null) {
+                String dimensions = configuration.getDimensions().toLowerCase().trim();
+                Matcher m = DIMENSIONS_PATTERN.matcher(dimensions);
+                if (m.matches()) {
+                    String width = m.group(1);
+                    String height = m.group(2);
+                    chromeOptions.addArguments(String.format("--window-size=%s,%s", width, height));
+                } else if (dimensions.equals("full") || dimensions.equals("fullscreen") || dimensions.equals("max")) {
+                    chromeOptions.addArguments("--window-size=1920,1080"); // workaround till a better way is found.
```

Comment:
just log warning that it si not possible to set fullscreen for headless, that user has to define specific dimensions.